doc/changes/DM-40906.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bug preventing bps cancel from working
```

Comment:
```suggestion
Fixed bug preventing ``bps cancel`` from working.
```

Reply:
If we are going to backport this it's probably best to not have this fragment and add it directly to the CHANGES.rst file for v26. That would mean that when you backport it we end up with a v26 release notes that matches both places.